src/controllers/expense.controller.js

```diff
@@ -0,0 +1,148 @@
+'use strict';
+
+const expenseService = require('../services/expense.service.js');
+const userService = require('../services/user.service.js');
+const { getFilteredExpenses } = require('../utils/getFilteredExpenses.js');
+
+const getAll = async(req, res) => {
+  const {
+    userId: recivedUserId,
+    categories,
+    from,
+    to,
+  } = req.query;
+  const userId = Number(recivedUserId);
```

Comment:
Why change the name of the user ID, only to later call the variable the same name as it was?